system/jlib/jtrace.cpp

```diff
@@ -171,6 +171,7 @@ class CTraceManager : implements ITraceManager, public CInterface
     CTraceManager();
     IMPLEMENT_IINTERFACE
 
+    virtual ISpan * createServerSpan(const char * name, const IProperties * httpHeaders, SpanFlags flags, std::chrono::nanoseconds spanStartTime) const override;
```

Comment:
comment: Another solution would be to add a default parameter to the standard constructor.  E.g.

std::chrono::nanoseconds spanStartTime = 0

which would avoid a proliferation of constructors.

Reply:
I actually started this way but thought you wouldn't appreciate the special default value.